src/main/webapp/editor/blocklyc.jsp

```diff
@@ -23,6 +23,8 @@
         <meta charset="utf-8">
         <meta name="base" content="<url:getUrl url="/"/>">
         <title>BlocklyProp</title>
+        <script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script>
```

Comment:
We need to pull this package into the webapp/cdn/lib folder to eliminate the off-site reference.